Header header
string name
float32 confidence
int32 x
int32 y
int32 left
int32 lower
int32 right
int32 upper
geometry_msgs/Point[] contour_pts 
sensor_msgs/CompressedImage image